jdi-light/src/main/java/com/epam/jdi/light/elements/complex/table/CacheAll.java

```diff
@@ -10,13 +11,13 @@
 public class CacheAll<T> extends CacheValue<T> {
     public CacheAll() { }
     public CacheAll(JFunc<T> getRule) { setRule(getRule); }
-    private boolean gotAll = false;
+    private Safe<Boolean> gotAll = new Safe<>(() -> false);
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6397461)